x-pack/plugins/searchprofiler/server/types.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IRouter, Logger } from 'kibana/server';
+import { ElasticsearchPlugin } from 'src/legacy/core_plugins/elasticsearch';
```

Comment:
Is it OK to import from the `src` alias?

Reply:
Will update!